src/Scope.ts

```diff
@@ -835,6 +836,10 @@ export class Scope {
             let referencedFile = this.getFileByRelativePath(scriptImport.pkgPath);
             //if we can't find the file
             if (!referencedFile) {
+                //skip the default bslib file, it will exist at transpile time but should not show up in the program during validation cycle
+                if (s`${scriptImport.pkgPath}` === `source${path.sep}bslib.brs`) {
```

Comment:
Do you need `path.sep`? Does `scriptImport.pkgPath` need to be normalized?

Reply:
Yeah we need path.sep right now. Until #329 merges, pkgPath uses windows separators on windows and unix slashes everywhere else. 

I'll check on whether scriptImport is already normalized elsewhere, perhaps we can skip this normalization.

Reply:
yeah, scriptImport has already been normalized, so this is redundant. i removed it. good catch!